src/domain/authorizations/authorizations.js

```diff
@@ -78,8 +78,15 @@ async function forwardAuthorizationMessage (headers, transactionRequestId, paylo
       )
     }
 
-    const query = method === Enum.Http.RestMethods.GET ? '?' + (new URLSearchParams(payload).toString()) : ''
-    const fullUrl = `${endpoint}/authorizations/${transactionRequestId}${query}`
+    let fullUrl
+    if (method === Enum.Http.RestMethods.GET) {
```

Comment:
Minor nitpick: could this be changed to `switch` instead? I generally find that much more readable

Reply:
@rmothilal & @lewisdaly - Aye Aye - fixed